src/App.jsx

```diff
@@ -109,28 +108,28 @@ export default function App() {
   }
 
   //Components and its logic for rendering
-
-  const quizElements = quiz.map((qa, index) => {
+  const quizElements = quiz.map((question, index) => {
     return (
       <Quiz
         key={index}
         id={index}
-        question={qa.question}
-        choices={qa.choices}
-        correctAnswer={qa.correct_answer}
-        selectedAnswer={qa.selected_answer}
+        question={question.question}
+        choices={question.choices}
+        correctAnswer={question.correctAnswer}
+        selectedAnswer={question.selectedAnswer}
```

Comment:
I think `question` is a better variable name than `qa` here, but that's just personal preference. I'd guess `qa` means 'Question and Answer', which is also an accurate name.